### DISCUSSION TITLE
INSTALLATION TUTORIAL PLEASE!!!!!!!!!!

### DISCUSSION THREAD

**Discussion by 1Kaux** [gennariarmando/v-hud#96] [Q&A] [answered]
Let's see, smart man, how am I supposed to understand that about the ASI loader when there isn't a single tutorial that explains how to use and install the mod correctly, step by step.
I would be very grateful if you could make a video on how to install step by step as it says on your website, since I don't understand anything about it and I suppose there are more people like me looking for a tutorial

**Comment by gennariarmando** [comment, marked as answer]
https://github.com/gennariarmando/v-hud#installation
Read this carefully.

Edit:
If you can't read, there are plenty of tutorials on youtube
https://www.youtube.com/watch?v=CenzcGp9tcQ
